packages/gatsby-recipes/src/cli.js

```diff
@@ -354,6 +350,33 @@ module.exports = async ({ recipe, graphqlPort, projectRoot }) => {
         }),
       ],
     })
+    const Plan = ({ state }) => {
+      // console.log(state)
+      const { exit } = useApp()
+      // Exit the app after we render
+      useEffect(() => {
+        exit()
+      }, [])
+
+      return (
+        <>
+          <ResourceProvider
+            value={
+              state.context.plan?.filter(p => p.resourceName !== `Input`) || []
+            }
+          >
+            <MDX key="DOC" components={components} remarkPlugins={[removeJsx]}>
+              {state.context.steps.join(`\n`)}
+            </MDX>
+            <Text>{`\n------\n`}</Text>
+            <Text color="yellow">To install this recipe, run:</Text>
+            <Text>{` `}</Text>
+            <Text>{`  `}gatsby recipes ./test.mdx --install</Text>
```

Comment:
This should be dynamic based on the passed recipe.

Reply:
fixed this